Makefile

```
@@ -166,7 +124,15 @@ $(plugin_k_include)/%: $(PLUGIN_SUBMODULE)/plugin/%
 	@mkdir -p $(dir $@)
 	install $< $@
 
-plugin-deps: $(plugin_includes) $(plugin_c_includes)
+
+$(KEVM_LIB)/%: $(PLUGIN_SUBMODULE)/build/%
+	@mkdir -p $(dir $@)
+	cp -r $< $@
+
+$(PLUGIN_SUBMODULE)/build/lib%:
+	cd $(PLUGIN_SUBMODULE) && make $*
```

This [causes a `No rule to make target 'LIBRARYNAME'`](https://github.com/runtimeverification/evm-semantics/actions/runs/5136360689/jobs/9243059585?pr=1865#step:5:22) because the pattern cuts of the `lib` prefix.
Easiest fix is probably to put the `lib` prefix back, since we want to only consider the targets files that start with `lib` here.
```suggestion
$(PLUGIN_SUBMODULE)/build/lib%:
	cd $(PLUGIN_SUBMODULE) && make lib$*
```